test/metrics/NumberMethods/test_number_methods.py

```diff
@@ -0,0 +1,55 @@
+# The MIT License (MIT)
+#
+# Copyright (c) 2020 Aibolit
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+import os
+from unittest import TestCase
+from aibolit.metrics.NumberMethods.NumberMethods import NumberMethods
+from pathlib import Path
+
+
+class TestCognitive(TestCase):
+    dir_path = Path(os.path.realpath(__file__)).parent
+    metric_nm = NumberMethods()
+
+    def test1(self):
+        lines = self.metric_nm.value(Path(self.dir_path, '1.java'))
```

Comment:
```suggestion
        metric = self.metric_nm.value(Path(self.dir_path, '1.java'))
```
And everywhere else.

Reply:
ok